src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -70,7 +71,26 @@ String topAttributeName(ConnectorConfig config) {
     @Override
     public void start(Map<String, String> props) {
         config = new ConnectorConfig(props);
-        client = new AmazonDynamoDBClient();
+
+            // TODO : Verify functionality of 
+            //      SystemPropertiesCredentialsProvider
+        if (config.accessKeyId == null  ||  config.secretKeyId == null) {
+            SystemPropertiesCredentialsProvider cp = new SystemPropertiesCredentialsProvider();
+            client = new AmazonDynamoDBClient(cp.getCredentials());
+            log.debug("AWS credentials from SystemProperties: {} {}",
+                    cp.getCredentials().getAWSAccessKeyId().toString(),
+                    cp.getCredentials().getAWSSecretKey().toString());
+        } else if (config.accessKeyId.value().length() == 0  ||  config.secretKeyId.value().length() == 0) {
+            client = new AmazonDynamoDBClient();
+            log.debug("AmazonDynamoDBClient created with default properties");
+        } else {
+            BasicAWSCredentials awsCreds = new BasicAWSCredentials(config.accessKeyId.value(), config.secretKeyId.value());
+            client = new AmazonDynamoDBClient(awsCreds);
+            log.debug("AWS credentials from connector configuration: {} {}",
```

Comment:
This will always log the password values as `[hidden]` (the implementation of `Password.toString()`), so don't think there is any use in including them here.